contracts/protocol/facets/SellerHandlerFacet.sol

```diff
@@ -52,17 +53,20 @@ contract SellerHandlerFacet is SellerBase {
     }
 
     /**
-     * @notice Updates a seller, with the exception of the active flag.
-     *         All other fields should be filled, even those staying the same.
+     * @notice Updates treasury address, if changed. Puts admin, operator, clerk and AuthToken in pending state, if changed.
```

Comment:
Since the protocol has several formal state machines, I would use the word “queue” rather than “state” to avoid any confusion.

Reply:
renamed